dflib/src/main/antlr4/org/dflib/ql/antlr4/Exp.g4

```diff
@@ -16,6 +16,15 @@ import org.dflib.*;
 import static org.dflib.ql.antlr4.ExpParserUtils.*;
 }
 
+@members {
+// global state of the parser
+PositionalParamSource paramSource = new PositionalParamSource();
+
```

Comment:
Not a big deal either way, but is it possible to pass parameters in `expRoot()`, `expSingle()`, `expArray()`?